pipelines/performance-tests-pipeline.yml

```diff
@@ -1000,13 +1033,15 @@ jobs:
             # Scale apps up
             kubectl scale deployment action-scheduler --replicas=1
             kubectl scale deployment action-worker --replicas=$ACTION_WORKER_REPLICAS
+            kubectl scale deployment action-processor --replicas=1
```

Comment:
Would it be a good idea to have a configurable replica amount for the action-processor, like we do for the action-worker and case-processor?

Reply:
I doubt we'll need more than one but I can look into it

Reply:
made it configurable